src/main/java/sonarquberepair/Main.java

```diff
@@ -8,39 +8,22 @@
 
 public class Main {
 
-	public static void repair(String pathToFile, String projectKey, int ruleKey) throws Exception {
-		Launcher launcher = new Launcher(){};
-		launcher.getEnvironment().setPrettyPrinterCreator(() -> {
-					SniperJavaPrettyPrinter sniper = new SniperJavaPrettyPrinter(launcher.getEnvironment());
-					sniper.setIgnoreImplicit(false);
-					return sniper;
-				}
-		);
+	public static void repair(String pathToFile, String projectKey, int ruleKey, boolean sniperMode) throws Exception {
```

Comment:
prefer an enum over a boolean: 1) it gives more readable code 2) it allows for backward compatible extensions in the future

```
enum PrettyPrintingStrategy {
  NORMAL,
  SNIPER,
}
```

Reply:
Great! Done.